src/components/Breadcrumb/Breadcrumb.spec.js

```diff
@@ -8,100 +8,100 @@ describe('Breadcrumb Suite', () => {
+        expect($(staticText).$('..').$('..').getAttribute('href')).not.toBeNull();
+    });
 
-    // it('Editable text header should be editable, but not contain a link', () => {
-    //     navigateToStory(component, childStories[2]);
-    //     expect($(editButton)).not.toBeNull();
-    //     expect($(editableText).$('..').getAttribute('href')).toBeNull();
-    // });
+    it('Editable text header should be editable, but not contain a link', () => {
+        navigateToStory(component, childStories[2]);
+        expect($(editButton)).not.toBeNull();
+        expect($(editableText).$('..').getAttribute('href')).toBeNull();
+    });
 
-    // it('Editable text header should be editable, and does a link', () => {
-    //     navigateToStory(component, childStories[3]);
-    //     expect($(editButton)).not.toBeNull();
-    //     expect($(editableText).$('..').getAttribute('href')).not.toBeNull();
-    //     expect($(editableText).$('..').getAttribute('class')).toContain('underlineOnHover');
-    // });
+    it('Editable text header should be editable, and does a link', () => {
+        navigateToStory(component, childStories[3]);
+        expect($(editButton)).not.toBeNull();
+        expect($(editableText).$('..').getAttribute('href')).not.toBeNull();
+        expect($(editableText).$('..').getAttribute('class')).toContain('underlineOnHover');
+    });
 
-    // it('Only clicking the edit icon displays the edit input field', () => {
-    //     executeInAllStories(component, [childStories[2], childStories[3]], () => {
-    //         $(editableText).waitForVisible(true);
-    //         $(editableText).click();
-    //         expect($(editableTextInput).isVisible()).toBe(false);
-    //         $(editButton).click();
-    //         $(editableTextInput).waitForVisible(constants.wait.short);
-    //     });
-    // });
+    it('Only clicking the edit icon displays the edit input field', () => {
+        executeInAllStories(component, [childStories[2], childStories[3]], () => {
+            $(editableText).waitForDisplayed(true);
+            $(editableText).click();
+            expect($(editableTextInput).isDisplayed()).toBe(false);
+            $(editButton).click();
+            $(editableTextInput).waitForDisplayed(constants.wait.short);
+        });
+    });
 
-    // it('Text input field should have a save a clear button', () => {
-    //     executeInAllStories(component, [childStories[2], childStories[3]], () => {
-    //         $(editableText).waitForVisible(constants.wait.short);
-    //         browser.moveToObject(editableText);
-    //         $(editButton).click();
-    //         expect($(editableTextInput).isVisible()).toBe(true);
-    //         expect($(saveEditButton).isVisible()).toBe(true);
-    //         expect($(clearEditButton).isVisible()).toBe(true);
-    //         $('body').click();
-    //     });
-    // });
+    it('Text input field should have a save a clear button', () => {
```

Comment:
maybe we can update the title to be `text input field should have a save and clear button`

Reply:
The breadcrumb spec has not been updated yet. This is still a work in progress. Not even sure what that change is. It looks like an accident as that "it" statement does not make sense.